shared/rocroller/lib/include/rocRoller/Operations/BlockScale.hpp

```diff
@@ -78,5 +78,29 @@ namespace rocRoller
             template <typename T1, typename T2, typename T3>
             friend struct rocRoller::Serialization::MappingTraits;
         };
+
+        class SubTileTranspose : public BaseOperation
+        {
+        public:
+            SubTileTranspose() = delete;
+
+            explicit SubTileTranspose(OperationTag input, std::vector<size_t> tileDimensions);
+
+            std::unordered_set<OperationTag> getInputs() const;
+            std::string                      toString() const;
+            const std::vector<size_t>&       tileDimensions() const;
```

Comment:
nit, for consistency:
```suggestion
            std::vector<size_t> const&       tileDimensions() const;
```